projects/canopy/src/lib/data-point/data-point/data-point.component.scss

```diff
@@ -1,6 +1,6 @@
 .lg-data-point {
   display: flex;
-  flex: 1;
```

Comment:
Good spot, I totally forgot this quirk, there might be some others floating around in there too.

Reply:
A quick scan shows that this is the only place we used the shorthand with the single value, so the other should be fine